public/app/core/components/AppChrome/AppChromeService.tsx

```diff
@@ -153,6 +154,13 @@ export class AppChromeService {
     if (isSamePath) {
       entries[0] = newEntry;
     } else {
+      if (lastEntry && lastEntry.name === newEntry.name) {
+        logDuplicateUnifiedHistoryEntryEvent({
+          entryName: newEntry.name,
+          lastEntryName: lastEntry.name,
+          newEntryURL: newEntry.url,
```

Comment:
presumably both name are the same (since it's the `if` condition) maybe we log the name + both URL there instead, to understand how they differ?

Reply:
good catch!